release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -695,6 +695,11 @@ Node disruption policies for Google Cloud Platform (GCP) clusters has been promo
 [id="ocp-4-17-machine-management_{context}"]
 === Machine management
 
+[id="ocp-4-17-enhancements-aws-mapi-placement_{context}"]
+==== Supporting AWS Placement Group Partition Number
+
+The {product-title} release {product-version} introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```

Comment:
The beginning here can be simplified since it's going right in the 4.17 RNs:
```suggestion
This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```